packages/@aws-cdk/aws-lambda-event-sources/lib/stream.ts

```diff
@@ -45,6 +70,7 @@ export interface StreamEventSourceProps {
    */
   readonly maxRecordAge?: Duration;
 
+
```

Comment:
Why the extra new line here? I think there are 4 places in this section that have 2 new lines, ideally you help me get rid of them all.